### PR TITLE
add charybdis nixos module

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: nixos/modules/services/networking/charybdis.nix [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.charybdis = {
+
+      enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/networking/charybdis.nix]: Please use `mkEnableOption`.

File: pkgs/servers/irc/charybdis/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "--enable-epoll"
+    "--enable-ipv6"
+    "--enable-openssl=${openssl}"
+    "--with-program-prefix=charybdis-"
+  ];
+
+  buildInputs = [ bison flex openssl ];
+
+  meta = {
+    description = "an extremely scalable ircd with some cooperation with the ratbox and ircu guys";
```
Comment (jagajaga) [inline pkgs/servers/irc/charybdis/default.nix]: Start with a capital letter and end with a dot, please.

Comment (copumpkin) [inline pkgs/servers/irc/charybdis/default.nix]: No dot, according to [the nixpkgs manual](http://nixos.org/nixpkgs/manual/#idm140737319009616):

> description
> A short (one-line) description of the package. This is shown by nix-env -q --description and also on the Nixpkgs release pages.
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.
> Wrong: "libpng is a library that allows you to decode PNG images."
> Right: "A library for decoding PNG images"

Comment (jagajaga) [inline pkgs/servers/irc/charybdis/default.nix]: Oh, yeah, sure! That was too late and I had to go to sleep :(
So just change the beginning.

Comment (Lassulus) [inline pkgs/servers/irc/charybdis/default.nix]: alright, changed it.

**Pull request merged**